helphours/templates/join.html

```diff
@@ -30,7 +31,7 @@ <h2 class="form-title">Join the Queue</h2>
             {% endif %}
 
             <div>
-                {{ form.submit(class="form-submit") }}
+                    {{ form.submit(class="form-submit", id="submit-button") }}
```

Comment:
```suggestion
                {{ form.submit(class="form-submit", id="submit-button") }}
```
Why is this extra indented?